planner/core/errors.go

```diff
@@ -61,6 +62,14 @@ const (
 	codeWindowNoRedefineOrderBy         = mysql.ErrWindowNoRedefineOrderBy
 	codeWindowDuplicateName             = mysql.ErrWindowDuplicateName
 	codeErrTooBigPrecision              = mysql.ErrTooBigPrecision
+	codeWindowFrameStartIllegal         = mysql.ErrWindowFrameStartIllegal
+	codeWindowFrameEndIllegal           = mysql.ErrWindowFrameEndIllegal
+	codeWindowFrameIllegal              = mysql.ErrWindowFrameIllegal
+	codeWindowRangeFrameOrderType       = mysql.ErrWindowRangeFrameOrderType
+	codeWindowRangeFrameTEMPORALType    = mysql.ErrWindowRangeFrameTEMPORALType
```

Comment:
Do we need to capitalize all characters in `temporal`?

Reply:
Oh, we do not need to...